src/common/internet.rs

```diff
@@ -51,13 +54,14 @@ pub enum InetAddr {
     IPv6(Ipv6Addr),
     #[cfg(feature = "tor")]
     Tor(TorPublicKeyV3),
```

Comment:
Didn't want to change this name right now because that might require some changes in some other places. I feel it would be easier for review to that in a separate commit.

Reply:
Separate commit will work! I usually use Jetbrains IDE "refactor" feature which instantly renames the variable/field/method throughout the whole project

Reply:
Huh, that's cool. I need to see if I can do that with vscode. I guess I will make the changes in a separate commit then.